### PR TITLE
Test PR.

### PR DESCRIPTION
**Pull request by fwalch**
Test for ReviewNinja.

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="http://app.review.ninja/fwalch/neovim/pull/14" target="_blank">ReviewNinja

File: src/nvim/version.c [hunk 1]
```diff
@@ -1046,6 +1046,15 @@ void intro_message(int colon)
     N_("by Bram Moolenaar et al."),
     N_("Vim is open source and freely distributable"),
     "",
+    N_("First time using a vi-like editor?"),
+    N_("Type :Tutor<Enter> to get started!"),
+    "",
+    N_("Already know your way around Vim?"),
```
Comment (fwalch) [inline src/nvim/version.c]: I don't like this text! You need to !fix this.

Comment (fwalch) [inline src/nvim/version.c]: I !fixed this.

**Comment by fwalch** [comment]
This looks :star:

**Pull request closed without merging**